File: examples/trance.py
# Lock into the zone.

import math

class Vector2:
  def __init__(self, x: float, y: float):
    self.x = x
    self.y = y

  def magnitude(self) -> float:
    return math.sqrt(self.x * self.x + self.y * self.y)

v = Vector2(3.0, 4.0)

magnitude = v.magnitude()

print(f"Magnitude: {magnitude}")

File: examples/trance.ts
// Lock into the zone.

class Vector2 {
  x: number;
  y: number;

  constructor(x: number, y: number) {
    this.x = x;
    this.y = y;
  }

  magnitude(): number {
    return Math.sqrt(this.x * this.x + this.y * this.y);
  }
}

const v = new Vector2(3.0, 4.0);

const magnitude = v.magnitude();

console.log(`Magnitude: ${magnitude}`);
